master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md

```diff
@@ -25,7 +25,7 @@ You must have a cluster which meets the following requirements:
 To install Calico, ensure you have a cluster which meets the above requirements and run the following command:
 
 ```
-kubectl apply -f http://docs.projectcalico.org/{{page.version}}/getting-started/kubernetes/installation/hosted/k8s-backend/calico.yaml
+kubectl apply -f {{site.url}}/{{page.version}}/getting-started/kubernetes/installation/hosted/k8s-backend/calico.yaml
```

Comment:
baseurl is blank. It's used if you're hosting multiple projects on the same site.